mod/metrics/set/type/category_value.rb

```diff
@@ -19,6 +19,10 @@ def pretty_values
   json_options? ? raw_values_from_hash : raw_value
 end
 
+def content= *args
+  super
```

Comment:
What does this?

Reply:
good catch. that was for debugging.  removing.